server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/GetByProjectAction.java

```diff
@@ -61,32 +60,33 @@ public void define(WebService.NewController context) {
     WebService.NewAction action = context.createAction(ACTION_GET_BY_PROJECT)
       .setInternal(true)
       .setSince("6.1")
-      .setDescription("Get the quality gate of a project.<br> " +
-        "Either project id or project key must be provided, not both.")
+      .setDescription("Get the quality gate of a project.")
```

Comment:
the permission description must be updated with the format: `Requires one of the following permissions:<br>…`